### ISSUE TITLE
Possible conflict with SameDifferent's More Villagers

### ISSUE THREAD

**Issue by Gamemassa** [henkelmax/trade-cycling#24]
### Bug description

Not entirely sure if either Trade Cycling or More Villagers is causing it (or something else), but when I click on villagers, I don't see the cycle button.

### Minecraft version

1.20.4

### Mod version

1.20.4-1.0.7

### Mod loader and version

Fabric Loader 0.15.7

### Steps to reproduce

Load game with both More Villagers and Trade Cycling and open a villager's trade window.

### Log files

N/A

**Comment by henkelmax** [comment]
Can you send a screenshot of the trading GUI?
You forgot to include your logs.
Does the cycle button work?

**Comment by henkelmax** [comment]
I just checked out the More Villagers mod and there's no version available for 1.20.4.

**Comment by Gamemassa** [comment]
> I just checked out the More Villagers mod and there's no version available for 1.20.4.

The latest version of More Villagers is working for 1.20.4 (even though it doesn't say it on the mod page).

**Comment by Gamemassa** [comment]
> Can you send a screenshot of the trading GUI? 

You forgot to include your logs.

- It wasn't causing a crash so not sure how the logs would help.

Does pressing the cycle keybind work?

- Yes, the Cycle keybind still works.

**Comment by henkelmax** [comment]
> The latest version of More Villagers is working for 1.20.4 (even though it doesn't say it on the mod page).

That might be the issue. You shouldn't use mods for versions its not supported for

**Comment by Gamemassa** [comment]
> > The latest version of More Villagers is working for 1.20.4 (even though it doesn't say it on the mod page).
> 
> That might be the issue. You shouldn't use mods for versions its not supported for

Actually, as of a minute ago, the issue has apparently fixed itself. The cycle button is now visible on both vanilla and modded villagers. I have absolutely no idea what caused the bug in the first place, or how I fixed it, but I did just make a new SP world because I added several more mods to my custom modpack (including almost all of YungNickYung's world-gen mods, which he just updated to 1.20.4) and all I can think is maybe something didn't load correctly in the previous world.
